packages/squiggle-lang/src/public/SqProject/ProjectItem.ts

```diff
@@ -225,13 +234,28 @@ export class ProjectItem {
         evaluate: asyncEvaluate,
       });
 
-      const bindings = contextAfterEvaluation.stack.asBindings();
       const exportNames = new Set(expression.value.value.exports);
-      const exports = bindings.filter((_, key) => exportNames.has(key));
+      const bindings = contextAfterEvaluation.stack
+        .asBindings()
+        .mapEntries(([key, value]) => {
+          let _value = value;
+          if (exportNames.has(key)) {
+            _value = value.mergeTags({
+              exportData: { sourceId: this.sourceId, path: [key] },
```

Comment:
This is more convoluted than I'd like, I see why you had to do it this way, but now I want to move this logic deeper in the reducer, store `exports` and `bindings` directly in context.

(I'll do it later)